install/install.go

```diff
@@ -265,19 +432,16 @@ func K8sInstaller(c *k8s.Client, o Options) error {
 
 	verify = o.Verify
 	var env string
-	if o.Env.Auto {
-		env = k8s.AutoDetectEnvironment(c)
-		if env == "none" {
-			if o.Save {
-				printMessage("⚠️\tNo env provided with \"--save\", setting env to  \"generic\"", true)
-				env = "generic"
-			} else {
-				return errors.New("unsupported environment or cluster not configured correctly")
-			}
+	env = k8s.AutoDetectEnvironment(c)
+	if env == "none" {
+		if o.Save {
+			printMessage("⚠️\tNo env provided with \"--save\", setting env to  \"generic\"", true)
+			env = "generic"
+		} else {
+			return errors.New("unsupported environment or cluster not configured correctly")
```

Comment:
Why was this done?
We could still want to save a legacy yaml without wanting to autodetect it.

Reply:
We have removed the env flag in this PR. We can have it enabled again if we want to give the user an option to generate kubearmor.yaml using the env they might provide.

Reply:
I have implemented manual env passing for --save.